api/tools/src/main/java/org/pmiops/workbench/tools/FetchWorkspaceDetails.java

```diff
@@ -9,20 +9,23 @@
 import org.apache.commons.cli.Options;
 import org.pmiops.workbench.db.dao.WorkspaceDao;
 import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.firecloud.FireCloudConfig;
+import org.pmiops.workbench.firecloud.FireCloudService;
+import org.pmiops.workbench.firecloud.FireCloudServiceImpl;
 import org.pmiops.workbench.firecloud.FirecloudTransforms;
 import org.pmiops.workbench.firecloud.api.WorkspacesApi;
 import org.pmiops.workbench.firecloud.model.FirecloudWorkspaceAccessEntry;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 
 /**
  * A tool that takes a Workspace namespace / Firecloud Project ID and returns details for any
  * workspaces found.
- *
- * <p>Details currently include... - Name - Creator Email - Collaborator Emails and Access Levels
  */
 @Configuration
+@Import({FireCloudServiceImpl.class, FireCloudConfig.class})
 public class FetchWorkspaceDetails {
```

Comment:
nit/tangent: I didn't know about this class (haven't worked with the tools much), but a couple of our other configs are named with a `SpringConfiguration` suffix (to distinguish from other config classes). I navigate randomly so much that I forget to look at package names, though I'm getting better.

Reply:
Was this meant to be attached to the CommandLineToolConfig file?

I also got annoyed at that naming degeneracy at some point. We have many, many more Spring configs than other configs, so I'd probably bend towards adding a suffix to non-Spring config files if we did need to add clarity. (Something like "AppConfig" or "ConfigModel" might be clear.)

But I've mostly just accepted that there's probably no great solution here.

Reply:
I was confused mainly on why `FileWorkspaceDetails` seems to have `@Configuration` in it. But maybe all the tools work that way.

Reply:
Ah – yes, all the tools work this way. Technically, it's just a configuration that provides a @CommandLineRunner bean. Spring docs for that annotation:

```
Interface used to indicate that a bean should <em>run</em> when it is contained within a {@link SpringApplication}.
```

It's a bit roundabout, but it's definitely the most reliable setup we've had for CLI tools yet.